pkgs/applications/misc/marktext/default.nix

```
@@ -32,7 +32,7 @@ appimageTools.wrapType2 rec {
 
   meta = with lib; {
     description = "A simple and elegant markdown editor, available for Linux, macOS and Windows";
-    homepage = "https://marktext.app";
+    homepage = "https://github.com/marktext/marktext";
```

```suggestion
    homepage = "https://www.marktext.cc";
```
based on the link in the repo, this is the new homepage being used

---

I prefer the GitHub URL since it's less likely to break again in the future.

Also concerning that https://marktext.cc doesn't redirect to the www version. Would be more interested in using upstream's domain if DNS was configured properly.

---

> I prefer the GitHub URL since it's less likely to break again in the future.

i can understand this, but honestly i feel this could be used for most projects in nixpkgs really. i feel we should just trust upstream here

> Also concerning that https://marktext.cc/ doesn't redirect to the www version.

definitely, and i would recommend making an issue with upstream here. however, as we are linking the www version directly this isn't going to really affect nixpkgs users

---

Upstream issue: https://github.com/marktext/marktext/issues/3806